bears/generate_package.py

```diff
@@ -128,10 +178,16 @@ def main():
                                  'LICENSE': str(bear_object.LICENSE),
                                  'LONG_DESCRIPTION': str(bear_object.__doc__),
                                  'BEAR_NAME': bear_name}
-
-            create_file_from_template(os.path.join('bears', 'setup.py.in'),
-                                      os.path.join('bears', 'upload',
-                                                   bear_name, 'setup.py'),
+            template_dir = os.path.join('bears', 'templates')
```

Comment:
This would be soo much more readable if written like:
```
 +            create_file_from_template(
 +                os.path.join(template_dir, 'setup.py.in'),
 +                os.path.join('bears', 'upload', bear_name, 'setup.py')
 +                if not args.conda else os.path.join(args.conda, 'setup.py'),
```